src/renderer/shared/ui-kit/Field/Field.tsx

```diff
@@ -0,0 +1,14 @@
+import { type PropsWithChildren } from 'react';
+
+type Props = {
+  text: string;
+};
+
+export const Field = ({ text, children }: PropsWithChildren<Props>) => {
+  return (
+    <label className="flex w-full flex-col gap-y-2">
+      <span className="text-footnote font-medium text-text-tertiary">{text}</span>
```

Comment:
What about case when we have addition text on the right side? 
<img width="338" alt="image" src="https://github.com/user-attachments/assets/12c95e19-3e63-4a67-a20d-b22f23446c38">

Reply:
Hm, good question
Is it the only component with such `suffix` element?
Can introduce something like `suffix` prop that accepts `ReactNode`
@sokolova-an @johnthecat what you guys think?

Reply:
Sounds good to me

Reply:
Maybe you can just use flexbox for this particular case. There is no need to add additional prop